neonvm/controllers/virtualmachine_qmp_queries.go

```diff
@@ -566,6 +677,22 @@ func QmpGetMigrationInfo(virtualmachine *vmv1.VirtualMachine) (MigrationInfo, er
 	return result.Return, nil
 }
 
+func QmpCancelMigration(virtualmachine *vmv1.VirtualMachine) error {
+	mon, err := QmpConnect(virtualmachine)
+	if err != nil {
+		return err
+	}
+	defer mon.Disconnect()
+
+	qmpcmd := []byte(`{"execute": "migrate_cancel"}`)
```

Comment:
Have no answer now. Btw, in my migration load test I've used `post-copy : false` because

> the down side is that during the postcopy phase, a failure of either side or the network connection causes the guest to be lost.

Reply:
Ok. Should we disable post-copy migration then? Or what should the behavior be if someone deletes a migration object after it's switched to post-copy?

I found [this patch](http://patchwork.ozlabs.org/project/qemu-devel/patch/20171205065307.21853-27-peterx@redhat.com/), which indicates that cancelling a post-copy migration will _guaranteed_ break the VM, which seems like... undesirable behavior.

Reply:
post-copy is part of migration spec, we can set it to `false` as i did in migration load tests. I prefer do not remove ability do post-copy migration

Reply:
Ok. What should the behavior be if the migration object is deleted after it switches to post-copy mode?

Reply:
there are options:
- leave it as is (e.g. send migration cancel command)
- pause migration deletion until migration finish (that bad as  vm and migration cloud stuck in migrating state forever)

in any case for prod we should not use post copy migration now, but we should have room for experiments.

Reply:
That makes sense. Given all that, can we change the default to have post-copy disabled, making it opt-in rather than opt-out? It would still leave room for experimentation, but helps us avoid accidentally running into these issues

Reply:
default value changed to `false`